src/client/Client.cc

```diff
@@ -9569,6 +9571,11 @@ int64_t Client::_write(Fh *f, int64_t offset, uint64_t size, const char *buf,
   lat -= start;
   logger->tinc(l_c_wrlat, lat);
 
+  if (fpos) {
```

Comment:
I think fpos can't be zero here. just set f->pos to "offset + size" ?

Reply:
thanks for your comments, 
when offset >=0, fpos is zero, and we don't need to set  f->pos.  also offset  may be changed. 
  if (offset < 0) {
...
    offset = f->pos;
...
  }